util/parse/Cargo.toml

```diff
@@ -7,5 +7,10 @@ description = "Helpers for parsing, particularly, for use with Clap and similar"
 readme = "README.md"
 
 [dependencies]
+hex = { version = "0.4", optional = true }
```

Comment:
Is there a reason to make it optional? I believe we used it in no_std environments with `hex = { version = "0.4", default-features = false, features = ["alloc"] }`

Reply:
removed the hex feature and use hex always in `mc-util-parse`